server/index.js

```diff
@@ -1,11 +1,19 @@
-const serve = require('koa-static');
 const Koa = require('koa');
+const serve = require('koa-static');
+const { get } = require('koa-route');
 
 const app = new Koa();
 
-// $ GET /hello.txt
+const data = require('../src/data/summits.json');
```

Comment:
This is actually a good example of fighting with ESLint, and whether it’s helping or hurting. In general, it’s better to do as it says, but this case is different.

Here, we’re loading a file in-memory, before the endpoint. What if a user doesn’t visit the API endpoint? That means the server loaded something in memory it didn’t need.

There are actually 3 ways to modify ESLint behavior:
1. Modify `.eslintrc` (this becomes a global rule though)
2. Use `/* eslint-disable RULE */` (here, it’d be `/* eslint-disable global-require */`). This disables the rule in the current file until a `/* eslint-enable RULE */` is reached.
3. Use `// eslint-disable-next-line global-require`. This applies to the next line only.

In general I prefer #3 when I like the rule, but I’m making an exception because I’m smarter.

So, yeah, you only need this 1% of the time, but you get the best of both worlds: you get great linting but can override it when you need to!

Reply:
Lot's of good tips in here.  Thanks! 

What's the best practice from a version control standpoint on these recommendations?  Is it best to delete the commit before merging, modify the files and then add a new cleaned up commit?

Reply:
So for this, I ✅’d it so you can merge if you want to. From here, it’s your call. If I wasn’t happy with it, I would have requested changes.

But in terms of Git, you can make an additional commit to this branch specifying what you did. Like, `Remove koa-parse`, etc. And just push onto this PR.

When you’re happy with it, merge it!